src/pages/playlist/Playlist.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { PlaylistFolderCard } from "../../components";
+import { useVideo } from "../../context";
+
+const Playlist = () => {
+  const { playlist } = useVideo();
+  return (
+    <div className="video-container">
+      <h4 className="text-center">Playlists</h4>
+
+      <div className="video-grid">
+        {playlist.length > 0 &&
+          playlist.map((list) => (
```

Comment:
you can try optional chaining `?.` for `playlist` to avoid checking the length separately.

Reply:
Oh! Right... I'll make the changes